portfolio-stylesheet.css

```diff
@@ -0,0 +1,400 @@
+	margin: 0;
+	text-decoration: none;
+	color: #000000;
+}
+
+.right {
+	width: 33.3%;
+}
+
+/* ========================================================================
+	Home Page
+   ========================================================================
+*/
+
+header {
+	padding: 360px 0;
+	width: 100%;
+	background-image: url('header.jpg');
+	background-size: cover;
+	background-position: top;
+	padding-left: 15px;
+	padding-right: 15px;
+}
+
+.home h2 {
+	padding: 40px;
+	font-size: 60px;
+	font-family: "DIN Next LT";
+}
+
+.about-me {
+	background-color: #cef2f2;
+	height: 450px;
+	display: flex;
+	justify-content: center;
+}
+
+.about-me h3 {
+	font-size: 50px;
+	margin-left: auto;
+	margin-right: auto;
+	font-family: "DIN Next LT";
+}
+
+.websites {
+	background-color: #fafad4;
+	height: 450px;
+	display: flex;
+}
+
+.websites h4 {
+	font-size: 50px;
+	margin-left: auto;
+	margin-right: auto;
+	font-family: "DIN Next LT";
+}
+
+.contact-me {
+	background-color: #f4daf5;
+	height: 450px;
+	display: flex;
+}
+
+.contact-me h5 {
+	font-size: 50px;
+	margin-left: auto;
+	margin-right: auto;
+	font-family: "DIN Next LT";
+}
+
+.before-footer {
+	display: flex;
+	background-color: #FFFFFF;
+	height: 170px;
+	flex-direction: column;
+	align-items: center;
+}
+
+/* ========================================================================
+	'About Me' Page
+   ========================================================================
+*/
+
+.aboutme {
+	font-family: "DIN Next LT";
+	background-color: #f0f2f2;
+	width: 1920px;
+}
+
+.aboutme-header {
+	display: flex;
+	justify-content: center;
+}
+
+.aboutme-heading h1 {
+	font-size: 50px;
+	padding-left: 45px;
+}
+
+.top-aboutme img {
+	border-radius: 50%;
+	width: 40%;
+	margin: 0;
+}
+
+.top-aboutme {
+	/*padding: 15px;*/
+	display: flex;
+	flex-direction: row;
+	height: 400px;
+	width: 1000px;
+	margin: auto;
+	padding-top: 50px;
+}
+
+.top-aboutme p {
+	font-size: 25px;
+	padding-left: 45px;
+}
+
+.middle-aboutme {
+	display: flex;
+	flex-direction: column;
+	margin: auto;
+	padding-top: 100px;
+	align-items: center;
+	width: 100%;
+}
+
+.aboutheading-two h1 {
+	font-size: 50px;
+}
+
+.middle-aboutme .icons {
+	display: grid;
+	grid-template-columns: repeat(6, 1fr);
+	grid-gap: 30px;
+	grid-auto-flow: row;
+	justify-items: center;
+	border: transparent;
+	width: 100%;
+	margin: 0 auto;
+	font-size: 25px;
+	padding-left: 20px;
+	padding-right: 20px;
+}
+
+.icons p {
+	text-align: left;
+}
+
+.aboutheading-three h1 {
+	font-size: 50px;
+	padding-left: 50px;
+}
+
+.cv-details {
+	width: 400px;
+	padding-left: 50px;
+}
+
+/* ========================================================================
+	'Portfolio' page
+   ========================================================================
+*/
+
+.projects-header {
+	font-family: "DIN Next LT";
+	display: flex;
+	justify-content: center;
+	
+}
+
+.projects-header h1 {
+	font-size: 50px;
+}
+
+.webtable {
+	display: grid;
```

Comment:
when shrinking screen size the grid stays in exact same place so its not easy to see all projects, need to scroll right, could change grid template for smaller screens or use flexbox which will auto wrap the projects boxes to the next line when made smaller